python/lib/sift_py/ingestion/config/yaml/load.py

```diff
@@ -31,6 +32,25 @@ def read_and_validate(path: Path) -> TelemetryConfigYamlSpec:
     return _validate_yaml(raw_config)
 
 
+def load_report_templates(paths: List[Path]) -> List[ReportTemplateConfig]:
```

Comment:
I think we should start breaking this yaml module apart. It is starting to feel awkward to have this be the dumping ground for parsing different yaml specs.

I think what make sense to me is having a top-level module along-side `report_template` maybe called `yaml_specs` and having the specs and parsing logic be separated by domain i.e. `yaml_specs/report_template`, `yaml_specs/rule`, `yaml_spec/telemetry_config` or something like that. We don't have to do this for everything right now, but can we move report templates logic out of this module? We can do the same for rule for your rules PR. We'll move telemetry config stuff out later.

Reply:
Sure, I think that sounds good! I'll go ahead and create this for report templates and then we can update this for the other domains the next time we touch them.

Reply:
Updated, let me know how that looks!